services/net/extract-quotes/Config/ExtractQuotesOptions.cs

```diff
@@ -1,9 +1,28 @@
-
-using TNO.Kafka.Models;
 using TNO.Services.Config;
 
 namespace TNO.Services.ExtractQuotes.Config;
 
+/// <summary>
+/// Configuration for a specific LLM API endpoint (primary or fallback)
+/// </summary>
+public class ApiConfig
+{
+    public List<string> ApiKeys { get; set; } = new List<string>();
+    public string ModelName { get; set; } = string.Empty;
+    // API URL for this specific endpoint
+    public string ApiUrl { get; set; } = string.Empty;
+}
+
+/// <summary>
+/// Configuration options specifically for LLM usage
+/// </summary>
+public class LLMOptions
```

Comment:
Create a file for each class.  It makes it difficult to maintain when classes are mixed into a single file.